### ISSUE TITLE
increasing lr during minigptv2 finetune

### ISSUE THREAD

**Issue by DJLee68** [Vision-CAIR/MiniGPT-4#444]
hello!

I trained with the `minigptv2` finetune you shared, and I've noticed that the learning rate increases as the learning progresses.
We use `lr_sched: "linear_warmup_cosine_lr"`, and as in
https://github.com/Vision-CAIR/MiniGPT-4/blob/a431672c89f9c8fe4a6829c8c37f75724bb323aa/train_configs/minigptv2_finetune.yaml#L269-L270
when we finetune `minigptv2`, `min_lr` is higher than `init_lr`, resulting increasing lr during training.
Is this intentional or a typo?
If it's intended, I wonder why LR is designed to get bigger as it trains.

**Comment by junchen14** [comment]
thanks for finding this issue. This shall be an typo and we will update the code

**Comment by lckj2009** [comment]
> thanks for finding this issue. This shall be an typo and we will update the code

Could you please answer the question about "gpu_id 1". Minigptv2

**Comment by Yuancheng-Xu** [comment]
I wonder if this problem is solved? Could you update the training script?
